### PR TITLE
Reorder subcommands and arguments

### PR DESCRIPTION
**Pull request by susannasiebert**
This will make the order of commands/arguments more intuitive for the users during display of the help text.

### CODE REVIEW

**Comment by agraubert** [comment]
+1

File: pvacseq/lib/main.py [hunk 1]
```diff
@@ -45,6 +45,15 @@ def main(args_input = sys.argv[1:]):
                         default=21)
     parser.add_argument('-g', '--gene-expn-file', help='genes.fpkm_tracking file from Cufflinks')
     parser.add_argument('-i', '--transcript-expn-file', help='isoforms.fpkm_tracking file from Cufflinks')
+    parser.add_argument('--net-chop-method',
+                        choices=lib.net_chop.methods,
+                        help="NetChop prediction method to use (\"cterm\" for C term 3.0, \"20s\" for 20S 3.0).  Default: \"cterm\" (C term 3.0)",
+                        default=None)
+    parser.add_argument(
```
Comment (acoffman) [inline pvacseq/lib/main.py]: Is it intentional that the alignment/formatting here doesn't match the other arguments?

**Comment by acoffman** [comment]
+1 whitespace issue notwithstanding!

File: pvacseq/lib/main.py (hunk 1 above)
Comment (susannasiebert) [inline pvacseq/lib/main.py]: It's just the way it was when we originally adde the parameter. I'll make it more consistent.

**Pull request merged**